README.md

```diff
@@ -102,10 +102,13 @@ Make sure the `service` binary and the `glove.eif` file are in the same director
 will both be in `target/release`:
 
 ```shell
-target/release/service --address=<LISTEN> --proxy-secret-phrase=<SECRET PHRASE> --node-endpoint=<URL> dynamodb --table-name=<GLOVE TABLE>
+target/release/service --address=<LISTEN> --proxy-secret-phrase=<SECRET PHRASE> --node-endpoint=<URL> --subscan-api-key=<API KEY> dynamodb --table-name=<GLOVE TABLE>
```

Comment:
FYI @wawrzek on the flag for the Subscan api key. Even a free key has higher quotas than no key.

Reply:
Do we need to change anything in our test deployment?

Reply:
Yes, you should at least get a free subscan key and use it for the test env.